admin_guide/service_accounts.adoc

```diff
@@ -119,3 +119,15 @@ policyConfig:
   openshiftInfrastructureNamespace: openshift-infra
 ----
 ====
+
+To controls whether or not to allow a service account to reference any secret
```

Comment:
Maybe more prescriptive? 

Set this value to true to require pod secret references to be whitelisted by their service accounts.  Set this value to false to allow pods to reference any secret in the namespace.